### PR TITLE
feat: prefetch hook updated in lh page

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- Implementing one query file in lh was not resulting as expected in page performance.
- code updated to clean up

Before one query file
before_local

After one query file
Screenshot 2023-07-26 at 10 59 51

### CODE REVIEW

File: src/graphql/query/userInfo.graphql [hunk 1]
```diff
@@ -9,10 +9,4 @@ query userInfo {
 			balance
 		}
 	}
-	general {
-		loanFindingPage: uiExperimentSetting(key: "loan_finding_page") {
```
Comment (dyersituations) [inline src/graphql/query/userInfo.graphql:13]: If you haven't, please add yourself a reminder to remove these old keys from Kiva Admin after the next prod release.

Comment (roger-in-kiva) [reply src/graphql/query/userInfo.graphql:13]: Since no more required right now, they are removed from Kiva Admin

Comment (dyersituations) [reply src/graphql/query/userInfo.graphql:13]: Can the other experiment settings be removed as well?

Comment (roger-in-kiva) [reply src/graphql/query/userInfo.graphql:13]: Yeah, they were inactive... looks like we forgot to removed them while disabling experiments definitively.

File: src/pages/LoanFinding/LoanFinding.vue [hunk 2]
```diff
@@ -150,25 +144,12 @@ export default {
...
-
-				const recommendedLoansPromise = client.query({
-					query: flssLoansQueryExtended,
-					variables: prefetchedRecommendedLoansVariables
-				});
-
+			return client.query({
+				query: userInfoQuery
+			}).then(() => {
+				return client.query({ query: experimentAssignmentQuery, variables: { id: RECOMMENDED_REPLACEMENT_EXP_KEY } }); // eslint-disable-line max-len
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:151]: Can this query be ran with the `userInfoQuery` above?

File: src/pages/LoanFinding/LoanFinding.vue [hunk 3]
```diff
@@ -150,25 +144,12 @@ export default {
...
-					query: userInfoQuery,
-				});
-
-				const recommendedLoansPromise = client.query({
-					query: flssLoansQueryExtended,
-					variables: prefetchedRecommendedLoansVariables
-				});
-
+			return client.query({
+				query: userInfoQuery
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:149]: Why is this query ran first now?

File: src/graphql/query/userInfo.graphql (hunk 1 above)
Comment (dyersituations) [reply src/graphql/query/userInfo.graphql:13]: Please remove if it's safe to do so, and ensure prod behavior is as expected after removing (no new console errors, for example).

Comment (roger-in-kiva) [reply src/graphql/query/userInfo.graphql:13]: Yeah, there are not errors about this.

File: src/pages/LoanFinding/LoanFinding.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:151]: Oh yeah, this is a nice suggestion!

File: src/pages/LoanFinding/LoanFinding.vue (hunk 3 above)
Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:149]: It was first before the update, just to keep like that... but we can have first this

`return client.query({ query: experimentAssignmentQuery, variables: { id: RECOMMENDED_REPLACEMENT_EXP_KEY } });`

and then all others...

File: src/pages/LoanFinding/LoanFinding.vue [hunk 4]
```diff
@@ -171,6 +169,7 @@ export default {
 				});
 
 				return Promise.all([
+					client.query({ query: userInfoQuery }),
```
Comment (dyersituations) [inline src/pages/LoanFinding/LoanFinding.vue:172]: Is there a reason why the user info was queried first before?

Comment (roger-in-kiva) [reply src/pages/LoanFinding/LoanFinding.vue:172]: No reason... It was the first query implemented in the page. any suggestion about this? do you mind a reason to don't be first?

File: src/graphql/query/userInfo.graphql (hunk 1 above)
Comment (dyersituations) [reply src/graphql/query/userInfo.graphql:13]: What about the other experiment keys removed in this PR?

**Review by dyersituations — APPROVED** [review]

File: src/graphql/query/userInfo.graphql (hunk 1 above)
Comment (roger-in-kiva) [reply src/graphql/query/userInfo.graphql:13]: No errors with all the keys removed

**Pull request merged**